google-api-client/OVERVIEW.md

```diff
@@ -1,36 +1,45 @@
-# Google API Client Library for Ruby Docs
+# Monolithic Client for Google APIs (DEPRECATED)
 
-The Google API Client Library for Ruby is designed for Ruby
-client-application developers. It offers simple, flexible
-access to many Google APIs.
+This library includes REST clients for all Discovery-based Google APIs as of December 2020.
```

Comment:
Link "Discovery" to `https://developers.google.com/discovery` or similar?

Reply:
done